commenting_system/migrations/0003_test_data.py

```diff
@@ -33,13 +33,13 @@ def generate_post(ind):
     description_formatted = 'This is my #{} favorite place! chill vibes and beautiful sea.'.format(
         ind + 1)
 
-    return post(nameOfPoster=author_choices[ind], nameOfLocation=place_choices[ind],
+    return Post(nameOfPoster=author_choices[ind], nameOfLocation=place_choices[ind],
                 photoURL=URL_formatted, Description=description_formatted)
 
 
 class Migration(migrations.Migration):
     dependencies = [
```

Comment:
You should clean, rm db.sqlite3 file before running migration,
It will not generate auto files for fix db migrations

Reply:
But shouldn't I want to have another migration file I did this because otherwise I should have erase the initial migration file in order to make it to work, and in order to make the data_test file to work after the naming change, I hade to change it name and dependencies as I explained.

Reply:
I will try it, but logic is run on clean environment each time, so if data created, just create it on clean env
I got similar errors to @Shoval-Ovadia.
Please verify on clean db.sqlite3, that migration is working, it will fail in CI also

Reply:
> I will try it, but logic is run on clean environment each time, so if data created, just create it on clean env
> I got similar errors to @Shoval-Ovadia.
> Please verify on clean db.sqlite3, that migration is working, it will fail in CI also
Please avoid auto generated files, it is a practice with higher chance to fail CI.

Reply:
see this diff, it resolve all auto files :+1: #
https://pastebin.com/8k1RVf6Q w/o auto file, run it on clean db, and reapply make migrations

Reply:
What should I understand from this diff? It seems like a result of using the makemigrations command after the changes I made not sure how is this helpful, what am I missing here?

Reply:
see the changes, the work is done for you already
you could see rgw changes similar to git changed (-) -> (+)
You can save the file into name.diff and open it with this 
https://tools.knowledgewalls.com/online-patch-diff-viewer